### ISSUE TITLE
It is now possible to generate PDFs with the SVG images

### ISSUE THREAD

**Issue by deanwampler** [Data-and-Trust-Alliance/DPS#8]
Recent releases of Pandoc support PDF generation with the appropriate SVG plugin, etc.

Using MacOS, I installed Pandoc v3.5 from the [repo's releases](https://github.com/jgm/pandoc/releases/tag/3.5). Using `brew install pandoc` installs an older 2.X release. (I didn't confirm if the older version is "good enough"...)

Otherwise, I followed [these MacOS instructions](https://github.com/jgm/pandoc/blob/main/INSTALL.md#macos) to install other dependencies, e.g., `librsvg`, `basictex`, and the recommended fonts.

After this, I was able to generate a working `dps.pdf` with this command:

```shell
pandoc -o dps.pdf specification.md
```

I confirmed the PDF contains the two images.

**Comment by JohnMoehrke** [comment]
Yes. The reason I did not put in a PDF was simply because of the added plugin, and there was no defined demand.

**Comment by deanwampler** [comment]
Maybe, but my first thought when I visited the site was "why can't I download a PDF of the spec?" How about at least adding a PDF there?
